docs/build/index.md

```diff
@@ -87,13 +99,28 @@ Ethereum has a large and growing number of tools to help developers build, test,
 - [Github](https://github.com/ethers-io/ethers.js/)
 - [Documentation](https://docs.ethers.io/ethers.js/html/)
 
+### Drizzle *Front-end libraries that make writing application frontends easier and more predictable.*
+- [github](https://github.com/trufflesuite/drizzle)
+
+
 ### Web3-wrapper *Typescript alternative to Web3.js*
 - [Github](https://github.com/0xProject/0x-monorepo/tree/v2-prototype/packages/web3-wrapper)
 - [Documentation](https://0x.org/docs/web3-wrapper#introduction)
 
 ### Looking for other options?
 - [Ethereum Developer Tools List #Frontend-Ethereum-APIs](https://github.com/ConsenSys/ethereum-developer-tools-list#frontend-ethereum-apis)
 
+## Backend
```

Comment:
I really struggle with the term "Backend" here  - also having problems seeing why "Web3.py A Python implementation of Web3.js." is in Backend and Web3.js is frontend

Reply:
Organize it however you see fit.  I just copied the categories from the Consensys page.